mindsdb_sdk/chatbots.py

```diff
@@ -0,0 +1,176 @@
+from typing import List
+from mindsdb_sdk.utils.objects_collection import CollectionBase
+
+
+class Chatbot:
+    """
+    Represents a chatbot that can be managed within a project.
+    """
+
+    def __init__(self, api, project, data: dict):
+        self.api = api
+        self.project = project
+        self.name = data.get('name')
+        self.database_name = data.get('database')
+        self.agent_name = data.get('agent')
+        self.model_name = data.get('model_name')
+        self.is_running = data.get('is_running')
+
+    def __repr__(self):
+        return f"{self.__class__.__name__}({self.project.name}.{self.name})"
+
```

Comment:
Just an idea what methods could be useful: 
- maybe add 'start'/'stop' methods?
Also a user would be interested in last error message for chatbot. is it accessible via rest API?

Reply:
for start stop methos we cane change is_running.

but "last error message", i am unabe to get it clearly